tests/integration/launch_container.sh

```diff
@@ -107,6 +107,10 @@ if $is_llm; then
   if [[ "$platform" == *"inf2"* ]]; then
     total=80
   fi
+  if [[ "$platform" == *"trtllm-sq"* ]]; then
+    echo "extra sleep of 15 min for smoothquant calibration"
```

Comment:
could we save quantized artifact instead of doing this at runtime?

Reply:
I set this number conservatively as a threshold. For the test I added, it doesn't take that much time. From test logs attached, entire process - smoothquant calibration, partitioning., loading took less than 15min. 
This is the only test we have covering quantization path.